nucliadb/src/nucliadb/search/api/v1/ask.py

```diff
@@ -62,6 +63,15 @@ async def ask_knowledgebox_endpoint(
         "This is slower and requires waiting for entire answer to be ready.",
     ),
 ) -> Union[StreamingResponse, HTTPClientError, Response]:
+    current_user: NucliaUser = request.user
+    # If present, security groups from AuthorizationBackend overrides any
+    # security group of the payload
```

Comment:
Should we document this behavior in the /ask endpoint?

Reply:
i dont think so, but i'll explain on the document about security groups where i'll introduce the use of these new tokens. from a "normal" use of the ask endpoint that cannot happen.

if you're using any kind of authorization proxy, (this tokens is exactly this in some sense) providing security groups directly from the end-user requests doesn't make much sense. In fact, probably this parameter shouldn't be exposed at all if we had a "cloud only" docs. but thats another story.